feathr_project/feathr/client.py

```diff
@@ -700,8 +700,10 @@ def _get_kafka_config_str(self):
             """.format(sasl=sasl)
         return config_str
 
-    def get_features_from_registry(self,project_name):
+    def get_features_from_registry(self, project_name: str):
```

Comment:
Change the return type to a Dict[str, FeatureBase] ? see https://github.com/linkedin/feathr/pull/232/files#diff-6bac58e5a2ddd48b771dc23fdc033617dfa12b1ef25bbfe825806a161c543e52